Riot/Modules/Room/Location/RoomTimelineLocationView.swift

```diff
@@ -39,6 +39,7 @@ class RoomTimelineLocationView: UIView, NibLoadable, Themable, MGLMapViewDelegat
     
     private var mapView: MGLMapView!
     private var annotationView: LocationMarkerView?
+    private static var usernameColorGenerator = UserNameColorGenerator()
```

Comment:
Note: This a detail but we can use an instance var and inject the same instance from the outside for every `RoomTimelineLocationView`